addon/components/nrg-appbar.hbs

```diff
@@ -0,0 +1,38 @@
+<div class="ui menu main fixed inverted">
+  <div class="ui fluid container">
+    <div class="left menu">
+      {{#if @showBackArrow}}
+        <a
+          role="button"
+          class="ui item icon sidebar-toggle-button"
+          {{on "click" this.onBackArrowClick}}
+        >
+          <i class="arrow left icon"></i>
+        </a>
+      {{else}}
+        <a
+          role="button"
+          class="ui item icon sidebar-toggle-button"
+          {{on "click" this.onToggleSidebar}}
+        >
+          <i class="content icon"></i>
+        </a>
+      {{/if}}
+      <div class="header item title-item">
+        <span class="title-text">{{this.application.pageTitle}}</span>
```

Comment:
Can this be optionally passed in? Instead, the template would use `this.pageTitle` and would define the getter as follows:
```js
get pageTitle() {
  return this.args.pageTitle ?? this.application.pageTitle;
}
```
This would allow users to overwrite the title at the `NrgAppbar` component level, without having to use the `application` service or the `RoutePageTitleMixin`.